web/style/transvision.css

```diff
@@ -296,6 +296,15 @@ th {
     padding: 0.5em;
 }
 
+th.select_header::after {
+    content: "Select";
+    font-weight: normal;
```

Comment:
I'm not a huge fan: 
- The existing label is not centered anymore.
- It's not available when table collapses (maybe that's OK).
- The cursor should change on hover.
- The text should probably say "Select column" for more clarity.

What about adding a real element in the HTML (e.g. a `<span>`) and use absolute positioning. We could also re-use the copy icon, and use "Select column" as alt/title